### PR TITLE
refactor(ContractAddresses): Extract WETH definitions

### PR DESCRIPTION
**Pull request by pxrl**
ContractAddresses pre-dates the existence of the external constants repository, which now contains the "canonical" definitions for Across. It's preferable to inherit the WETH definition from there because it ensures that the WETH address is consistent throughout the bot. Additionally, it removes a step for adding a new chain.

### CODE REVIEW

File: src/dataworker/DataworkerUtils.ts [hunk 1]
```diff
@@ -542,8 +543,9 @@ export async function _buildPoolRebalanceRoot(
  * @param chainId chain to check for WETH and ETH addresses
  * @returns WETH and ETH addresses.
  */
-function getWethAndEth(chainId): string[] {
-  const wethAndEth = [CONTRACT_ADDRESSES[chainId].weth.address, CONTRACT_ADDRESSES[chainId].eth.address];
+function getWethAndEth(chainId: number): string[] {
+  // Can't use TOKEN_SYMBOLS_MAP for ETH because it parrots the WETH addresses, which is not correct for this use case.
```
Comment (nicholaspai) [inline src/dataworker/DataworkerUtils.ts:547]: What is "parrots"? Probably should use a more generic word

File: src/common/ContractAddresses.ts [hunk 2]
```diff
@@ -238,10 +215,6 @@ export const CONTRACT_ADDRESSES: {
       address: "0x4200000000000000000000000000000000000010",
       abi: OVM_L2_STANDARD_BRIDGE_ABI,
     },
-    weth: {
-      address: "0x4300000000000000000000000000000000000004",
```
Comment (nicholaspai) [inline src/common/ContractAddresses.ts:242]: Have you confirmed these addresses all match up with the other import?

**Review by nicholaspai — APPROVED** [review]
+1 nice cleanup

File: src/clients/bridges/ArbitrumAdapter.ts [hunk 3]
```diff
@@ -24,30 +25,31 @@ import { CCTPAdapter } from "./CCTPAdapter";
...
+  [TOKEN_SYMBOLS_MAP.USDC.addresses[MAINNET]]: "0x096760F208390250649E3e8763348E783AEF5562", // USDC
+  [TOKEN_SYMBOLS_MAP.USDT.addresses[MAINNET]]: "0x096760F208390250649E3e8763348E783AEF5562", // USDT
+  [TOKEN_SYMBOLS_MAP.WETH.addresses[MAINNET]]: "0x6c411aD3E74De3E7Bd422b94A27770f5B86C623B", // WETH
+  [TOKEN_SYMBOLS_MAP.DAI.addresses[MAINNET]]: "0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65", // DAI
+  [TOKEN_SYMBOLS_MAP.WBTC.addresses[MAINNET]]: "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // WBTC
+  [TOKEN_SYMBOLS_MAP.UMA.addresses[MAINNET]]: "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // UMA
+  [TOKEN_SYMBOLS_MAP.BADGER.addresses[MAINNET]]: "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // BADGER
+  [TOKEN_SYMBOLS_MAP.BAL.addresses[MAINNET]]: "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // BAL
+  [TOKEN_SYMBOLS_MAP.ACX.addresses[MAINNET]]: "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // ACX
+  [TOKEN_SYMBOLS_MAP.POOL.addresses[MAINNET]]: "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // POOL
```
Comment (pxrl) [inline src/clients/bridges/ArbitrumAdapter.ts:52]: This was just a driveby change that reduces a some duplication.

**Review by bmzig — APPROVED** [review]

File: src/common/ContractAddresses.ts (hunk 2 above)
Comment (pxrl) [reply src/common/ContractAddresses.ts:242]: Confirmed :heavy_check_mark: 

weth

File: src/dataworker/DataworkerUtils.ts (hunk 1 above)
Comment (pxrl) [reply src/dataworker/DataworkerUtils.ts:547]:
```
verb
repeat [mechanically]
"encouraging students to parrot back information"
```

Will change to `duplicates`.

Comment (pxrl) [reply src/dataworker/DataworkerUtils.ts]: Done: https://github.com/across-protocol/relayer/pull/1685/commits/3541a1abcfd8e4808e6d607a7d4c37ca18f93add
